### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.cognitiveservices.vision.face.models.APIErrorException
  fullName: azure.cognitiveservices.vision.face.models.APIErrorException
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - inheritance:
          - type: builtins.object
          type: builtins.BaseException
        type: builtins.Exception
      type: msrest.exceptions.ClientException
    type: msrest.exceptions.HttpOperationError
  langs:
  - python
  module: azure.cognitiveservices.vision.face.models
  name: APIErrorException
  source:
    id: APIErrorException
    path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\api_error.py
    remote:
      branch: master
      path: azure-cognitiveservices-vision-face\azure\cognitiveservices\vision\face\models\api_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 32
  summary: 'Server responsed with exception of type: ''APIError''.

    '
  syntax:
    content: APIErrorException(deserialize, response, *args)
    parameters:
    - description: 'A deserializer

        '
      id: deserialize
    - description: 'Server response to be deserialized.

        '
      id: response
  type: class
  uid: azure.cognitiveservices.vision.face.models.APIErrorException
references: []
